src/test/javascript/spec/component/programming-exercise/programming-exercise.component.spec.ts

```diff
@@ -187,15 +187,6 @@ describe('ProgrammingExercise Management Component', () => {
         expect(modalService.open).toHaveBeenCalledOnce();
     });
 
-    it('should open repo export modal', () => {
```

Comment:
This deletion causes an unused import at the top of the file

Reply:
Thanks missed that... Has now been removed 👍